misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-shopware6/src/main/java/de/metas/camel/externalsystems/shopware6/api/ShopwareClient.java

```diff
@@ -424,8 +428,8 @@ public JsonCurrencies getCurrencies()
 		{
 			final JsonOrder jsonOrder = objectMapper.treeToValue(orderJson, JsonOrder.class);
 
-			final JsonOrderAndCustomId.JsonOrderAndCustomIdBuilder jsonOrderAndCustomIdBuilder =
-					JsonOrderAndCustomId.builder()
+			final OrderCandidate.OrderCandidateBuilder orderCandidateBuilder =
+					OrderCandidate.builder()
 							.jsonOrder(jsonOrder);
 
 			if (customIdJSONPath != null)
```

Comment:
please check for "isNotblank" at all 3 custom-json-paths.
they are set in the UI and a user might accidentally drop some spaces.